folly/test/TraitsTest.cpp

```diff
@@ -98,6 +98,19 @@ TEST(Traits, unset) {
   EXPECT_TRUE(IsRelocatable<F4>::value);
 }
 
+TEST(Traits, triviallyMoveConstructible) {
+#pragma GCC diagnostic push
```

Comment:
Please use our wrapper macros in `folly/Portability.h` - this would otherwise fail MSVC builds.

An alternative option would be to make a new header `folly/test/TraitsTestHelpers.h` with this type, but that is higher-ceremony than this.

Reply:
Thanks, I just switched to using `FOLLY_CLANG_DISABLE_WARNING` in this test case only. I agree with you that this doesn't warrant the creation of a new header file.